test/unit/BCoWPool.t.sol

```diff
@@ -130,3 +148,84 @@ contract BCoWPool_Unit_EnableTrading is BaseCoWPoolTest {
     bCoWPool.enableTrading(appData);
   }
 }
+
+contract BCoWPool_Unit_Verify is BaseCoWPoolTest {
+  function test_Revert_NonBoundToken() public {
```

Comment:
There is a small bug in the verify function, it's checking the inToken both times. It's a good opportunity to test 2 separate cases, one for inToken and one for outToken.

![image](https://github.com/defi-wonderland/balancer-v1-amm/assets/114922365/533b812c-0f03-42a7-9b5c-b09d257fc386)